ecl/eclccserver/eclccserver.cpp

```diff
@@ -1115,6 +1195,72 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
     }
 };
 
+
+//---------------------------------------------------------------------------------------------------------------------
+
+class EclccLingeringCompiler
+{
+    const char * instanceName;
+    EclccCompiler compiler;
+    Owned<IJobQueue> queue;
+    unsigned lingerTimeMs;
+    double costPerHour;
+
+public:
+    EclccLingeringCompiler(IPropertyTree * globals, unsigned _idx)
+    : instanceName(globals->queryProp("@name")), compiler(instanceName, _idx)
```

Comment:
I know 'globals' is in scope in the context EclccLingeringCompiler is used.. but it looks unsafe to keep pointer to @name here, for clarify, future safety, is it worth storing in a StringAttr despite not being strictly being necessary at the moment?